_pages/kriskirov.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: kriskirov
+pageUrl: kriskirov.github.io
+timestamp: TODAYS-24-2017-04-Mo
+---
```

Comment:
Hi @kriskirov! A few changes that may need to happen before the tests pass are:
- Make sure the date is formatted like `timestamp: 2017-04-24` 
- Try adding an extra line break after line 5

Let me know if you want or need any help 😄